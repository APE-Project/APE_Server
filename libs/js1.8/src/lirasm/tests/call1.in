ptr = alloc 8
a = int 65
sti a ptr 0
b = int 66
sti b ptr 1
c = int 67
sti c ptr 2
zero = int 0
sti zero ptr 3
ss = icall puts cdecl ptr
ret ss
