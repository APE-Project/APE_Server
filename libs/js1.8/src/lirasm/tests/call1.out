ABC
Output is: 4
